src/main/java/net/sf/jabref/external/TransferableFileLinkSelection.java

```diff
@@ -43,7 +43,7 @@ public TransferableFileLinkSelection(BasePanel panel, List<BibEntry> selection)
         if (tm.getRowCount() > 0) {
             // Find the default directory for this field type, if any:
             List<String> dirs = panel.getBibDatabaseContext().getMetaData().getFileDirectory(Globals.FILE_FIELD);
-            File expLink = FileUtil.expandFilename(tm.getEntry(0).link, dirs);
+            File expLink = FileUtil.expandFilename(tm.getEntry(0).link, dirs).get();
```

Comment:
I would add here a isPresent check. (the previous code did not throw an exception if expandFilename returned null, but now it does throw an error in this case)